src/index.js

```diff
@@ -1,12 +1,44 @@
 import './styles/main.scss';
-import add from './add.js';
+import { add, getData } from './add.js';
 
 const submitBtn = document.getElementById('submit');
+// const clearinput = () => {
+//   let name = document.getElementById('name').value;
+//   let score = document.getElementById('score').value;
+//   name = '';
+//   score = '';
+// };
```

Comment:
- [ ] kindly remove this commented code from your code I don't think we need in future ?? do we